src/features/main_menu/mobile.vue

```diff
@@ -95,14 +95,15 @@
       </v-row>
       <div style="height: 40px" />
 
-      <v-footer color="primary" fixed>
+      <v-footer color="primary" fixed style="padding-bottom: calc(6px + env(safe-area-inset-bottom, 0px))">
```

Comment:
`6px` is the padding that Vuetify gives the footer component by default. I'd rather not have hard-coded this magic number, but I don't know enough about SASS to know if there's a way to extract just the vertical padding value from the variable documented here: https://vuetifyjs.com/en/api/v-footer/#sass-footer-padding